src/plugins/hathor_debug.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+let debugLong;
+
+/* istanbul ignore next */
+async function checkDeps() {
+  const requiredDeps = {
+    yargs: '^16.2.0',
+  };
+  await Promise.all(requiredDeps.map(async d => {
+    try {
+      return import(d);
+    } catch (e) {
+      throw new Error(`Some plugin dependencies are missing, to install them run:
+      $ npm install ${Object.entries(requiredDeps).map(x => [x[0], x[1]].join('@')).join(' ')}`);
+    }
+  }));
+}
+
+export const getSettings = () => {
+  const yargs = require('yargs/yargs'); // eslint-disable-line global-require
+  const { hideBin } = require('yargs/helpers'); // eslint-disable-line global-require
+  const { argv } = yargs(hideBin(process.argv));
+
+  debugLong = argv.plugin_debug_long || process.env.HEADLESS_PLUGIN_DEBUG_LONG;
+  return { debugLong };
+};
+
+export function eventHandler(data) {
+  const message = JSON.stringify(data);
+  if (message.length < 1000) {
+    console.log(message);
+    return;
+  }
+  switch (debugLong) {
```

Comment:
We don't have a default for this parameter, right? So if not configured, it will reach the `default` in this switch. Is that correct?

Reply:
Yes, the default behavior is to log a dict with the event type and wallet id but without the event data since it would generate too much data on the output logs.